GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -229,34 +229,36 @@ void StartAsyncDataLoad()
             {
                 var cancellationToken = _asyncLoadCancellation.Next();
 
-                ThreadHelper.JoinableTaskFactory.RunAsync(() => LoadLinksForRevisionAsync(_revision)).FileAndForget();
```

Comment:
It seems that LoadLinksForRevisionAsync does not block, so it could remain async.

Reply:
Yes and no. It isn't executing a git operation, you're right; but he parser may be reading definitions from the filesystem.
I'll update it.